encoding/ccf/ccf_test.go

```diff
@@ -13421,17 +13746,22 @@ func TestDecodeInvalidData(t *testing.T) {
 		},
 	}
 
-	for _, tc := range testCases {
+	test := func(tc testCase) {
 		t.Run(tc.name, func(t *testing.T) {
```

Comment:
Could be parallelized too?
```suggestion
		t.Run(tc.name, func(t *testing.T) {
			t.Parallel()
```